doc/implementation_details.rst

```diff
@@ -654,6 +654,18 @@ Name functions and variables using "snake_case", for example
 
 Pointer names start with ``p_``, for example ``p_data_status``.
 
+Instead of::
+
+   if (have_dhcp == true){...}
+   if (have_dhcp == false){...}
+
+use::
+
+   if (have_dhcp){...}
+   if (have_dhcp == false){...}
```

Comment:
if (!have_dhcp)         ?

If not, this example should be in the "instead of list"

Reply:
Done